package.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "engines": {
-    "node": "12.14.0"
+    "node": "^12.14.0 || ^13.5.0"
```

Comment:
```suggestion
    "node": "13.5.0"
```

Reply:
You'll need the `^`  to allow from `13.5.0` to `13.5.x` otherwise it will be too restrictive.